### ISSUE TITLE
Switch item doesn't work with accessibility

### ISSUE THREAD

**Issue by Johan-dutoit** [evetstech/react-native-settings-list#15]
I am having issues getting accessibility working with the switch item.  The surrounding `TouchableHighlight` doesn't allow focus on the `Switch` and thus cannot be tapped on.

Is there a workaround?

**Comment by Johan-dutoit** [comment]
https://github.com/evetstech/react-native-settings-list/pull/16
